src/Workspaces/Core/Portable/CodeGeneration/AbstractCodeGenerationService.cs

```diff
@@ -228,8 +228,8 @@ public async Task<Document> AddImportsAsync(Document document, CodeGenerationOpt
             }
 
             // Filter out the members that are implicitly declared.  They're implicit, hence we do
-            // not want an explicit declaration.
-            var filteredMembers = membersList.Where(m => !m.IsImplicitlyDeclared);
+            // not want an explicit declaration. The only exception are fields generated from implicit tuple fields.
+            var filteredMembers = membersList.Where(m => !m.IsImplicitlyDeclared || m.IsTupleField());
```

Comment:
>|| m.IsTupleField() [](start = 81, length = 19)

I'm not sure this is the best fix. I would prefer to re-create fields from tuple fields during the refactoring and change IsImplicitlyDeclared to true in fields. However, I could not find an infrastructure for doing this. Please point me if you see a better solution.